test/test_core/test_data_manipulation.py

```diff
@@ -258,6 +258,27 @@ def test_rescaling(self, create_rescaling):
     def test_additional_data(self, create_additional_data):
         return create_additional_data
 
+def test_calc_ori_knn(coordinates, ref_orientations):
```

Comment:
this test are not running. You need to use pytest (simply typing pytest on the console when you are on the folder)